app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
```

Comment:
- [ ] Good job working on the login and registration  but I think it would be great if you worked on it to make it like the wireframe as mentioned in the requirements since you should make it like the given sneak peek so kindly improve it to make it the same 💪🏻🔥🚀

- There  is no wireframe to the registration  but I think you could do it like the log in🙏🏻

![image](https://github.com/OsvaldoBC/blog/assets/58553711/c46181d5-93f6-477a-b9be-1c5b5d6caff8)


- Your
![image](https://github.com/OsvaldoBC/blog/assets/58553711/12d3ac72-d7dd-4713-be83-df3e91578bae)

- Given
![image](https://github.com/OsvaldoBC/blog/assets/58553711/1fa4b25e-e52f-4254-9d49-479d0431011f)